scripts/biomedical/STITCH/tmcf/drugs.tmcf

```diff
@@ -0,0 +1,21 @@
+Node: E:drug_info->E1
+typeOf: dcs:Drug
+dcid: C:drug_info->pubchem_dcid
+name: C:drug_info->pubchem_ID
```

Comment:
the name should be referring to the column name in the CSV and you should remove the property common name. The column pubchem_ID should be under the property stitchCompoundID.

Reply:
Thank you for the feedback. I see in Data Commons that the property 'name' usually shows the chembl ID. Since the 'name' column refers to the name of the compound,  I labeled the property as 'commonName'. Just to clarify, would you like me to replace "commonName: C:drug_info->name" with "name: C:drug_info->name"?